src/test/java/seedu/address/commons/util/CollectionUtilTest.java

```
@@ -107,6 +98,46 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    private void assertNullPointerExceptionThrown(Object... objects) {
+        try {
+            requireNonNull(objects);
+            fail();
+        } catch (NullPointerException npe) {
+        } catch (Exception e) {
+            fail();
+        }
+    }
+
+    private void assertNullPointerExceptionThrown(Collection<?> objects) {
+        try {
+            requireNonNull(objects);
+            fail();
+        } catch (NullPointerException npe) {
+        } catch (Exception e) {
+            fail();
+        }
+    }
+
+    private void assertNullPointerExceptionNotThrown(Object... objects) {
+        try {
+            requireNonNull(objects);
+            return;
+        } catch (NullPointerException npe) {
+            fail();
+        }
+        fail();
```

1. Do we need a try-catch block for this? Seems like you can just let the exception throw?
2. Even if we need try-catch, we don't need the `return;` on line 124 and `fail();` on line 128:

```
try {
    requireNonNull(objects);
} catch (NullPointerException npe) {
    fail();
}
```